Sources_v3/StreamChat/Controllers/CurrentUserController/CurrentUserController.swift

```diff
@@ -364,6 +364,53 @@ public extension _CurrentChatUserController {
         }
     }
     
+    /// Updates the current user data.
+    ///
+    /// By default all data is `nil`, and it won't be updated unless a value is provided.
+    ///
+    /// - Parameters:
+    ///   - name: Optionally provide a new name to be updated.
+    ///   - imageURL: Optionally provide a new image to be updated.
+    ///   - userExtraData: Optionally provide new user extra data to be updated.
+    ///   - completion: Called when user is successfuly updated, or with error.
+    func updateUserData(
+        name: String? = nil,
+        imageURL: URL? = nil,
+        userExtraData: ExtraData.User? = nil,
+        completion: ((Error?) -> Void)? = nil
+    ) {
+        guard let currentUserId = currentUser?.id else {
+            completion?(ClientError.CurrentUserDoesNotExist())
+            return
+        }
+        
+        let params: [Any?] = [name, imageURL, userExtraData]
+        guard !params.allSatisfy({ $0 == nil }) else {
+            log.warning("Update user request not performed. All provided data was nil.")
+            return
```

Comment:
You should call the completion here, too.

Reply:
Fixed 👍